cpp/simulations/abm.cpp

```diff
@@ -695,10 +696,13 @@ mio::IOResult<void> run(const fs::path& result_dir, size_t num_runs, bool save_s
     while (run_idx <= num_runs) {
         // Make a simulation using a copy from the original world
         auto sim = mio::abm::Simulation(t0, mio::abm::World(world));
+        // Add a time series writer to the simulation
+        mio::History<mio::abm::TimeSeriesWriter, mio::abm::LogInfectionState> historyTimeSeries{
+            Eigen::Index(mio::abm::InfectionState::Count)};
         // Advance the world to tmax
         sim.advance(tmax);
```

Comment:
Should this history be passed to advance?

Reply:
That is correct.